x-pack/plugins/monitoring/server/lib/standalone_clusters/has_standalone_clusters.ts

```diff
@@ -28,7 +28,7 @@ export async function hasStandaloneClusters(req: LegacyRequest, indexPatterns: s
           },
           {
             terms: {
-              'metricset.name': ['logstash_stats', 'logstash_state', 'beats_stats', 'beats_state'],
+              'event.dataset': ['logstash.node.stats', 'logstash.node', 'beat.stats', 'beat.state'],
```

Comment:
is there any reason we couldn't keep using `metricset.name` and update the values to be correct instead of using event.dataset?  I thought we had [agreed to use metricset.name](https://github.com/elastic/kibana/pull/119112#discussion_r770951534).